docker/2.0.0/py2/Dockerfile.cpu

```diff
@@ -113,7 +116,8 @@ RUN ${PIP} install --no-cache-dir -U \
     # the library version to be overwritten
     # ${PIP} install --no-cache-dir -U ${TF_URL} \
  && ${PIP} install --no-cache-dir -U \
-    $TENSORFLOW_WHL \
+    /tmp/$TENSORFLOW_WHL \
```

Comment:
You can directly do a pip install from the url without downloading and then deleting it after the installation.